Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -1451,6 +1455,18 @@ void AccessibilityNodeObject::visibleText(Vector<AccessibilityText>& textOrder)
         if (!text.isEmpty())
             textOrder.append(AccessibilityText(text, AccessibilityTextSource::Children));
     }
+
+    if(textOrder.size() == 0 && this->isTableRow()) {
```

Comment:
Just a question before merging: If I understand correctly, this covers some downstream-specific needs not intended for upstreaming into current webkit's master, right? Or would be something that could be added to the spec? Couldn't find the WAI-ARIA spec regarding this, though.

Reply:
Thanks for the clarification. I think this PR can be merged as is and we can open a bug upstream to add a test/implement this.